modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/model/impl/FragmentEntryImpl.java

```diff
@@ -74,6 +76,16 @@ public String getImagePreviewURL(ThemeDisplay themeDisplay) {
 		return StringPool.BLANK;
 	}
 
+	@JSON
+	@Override
+	public int getStatus() {
```

Comment:
If you doing this way I think you can drop the status column.  Otherwise, I think you should override the publish method and set the status there.

Reply:
Although that might be difficult if you need the WorkflowedModel interface for some reason.

Reply:
Thanks for the suggestion! Overriding the publish method and setting the status there is something that already came up during the review with @jkappler. We agreed to do that on a follow-up pull, since this one already has quite many commits.